Atomic/util.py

```diff
@@ -227,6 +234,7 @@ def skopeo_layers(image, args=[], layers=[]):
         args = ['skopeo', 'layers'] + args + [image] + layers
         r = subp(args, cwd=temp_dir)
         if r.return_code != 0:
+            is_v1_registry(image)
             raise ValueError(r.stderr.decode(sys.getdefaultencoding()))
     except OSError:
```

Comment:
Just an oversight as the second def was last touched by someone else.  Made the same now.